invenio_cli/cli/packages.py

```diff
@@ -67,11 +68,28 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages(packages)
+
+    on_fail = f"Failed to show outdated packages."
+    on_success = f"Outdated packages."
+
+    run_steps(steps, on_fail, on_success)
+
+    os.popen('pipenv update --outdated')
 
 
 @packages.command()
+@click.argument("package", nargs=1, type=str)
 @pass_cli_config
-def update(cli_config):
+def update(cli_config, package):
     """Update a single Python python package."""
```

Comment:
doubt: Can it update *all* outdated if `package` is not passed?

Reply:
I'm not sure I will have to check